src/runner.ts

```diff
@@ -201,13 +243,24 @@ export class ActionRunner {
     return matches;
   }
 
-  async runAction(inputs: Omit<Inputs, "repoToken">): Promise<boolean> {
+  /** Core runner of the app.
+   * 1. It fetches the config
+   * 2. It validates all the pull request requirements based on the config file
+   * 3. It generates a status check in the Pull Request
+   * 4. WIP - It assigns the required reviewers to review the PR
+   */
+  async runAction(inputs: Omit<Inputs, "repoToken">): Promise<CheckData> {
     const config = await this.getConfigFile(inputs.configLocation);
 
-    const success = await this.validatePullRequest(config);
+    const reports = await this.validatePullRequest(config);
+
+    this.logger.info(reports.length > 0 ? "There was an error with the PR reviews." : "The PR has been successful");
```

Comment:
This log line feels off. 
Judging byt the code of `generateCheckRunData`, we can have `reports > 0`, while still having enough reviews to mark the PR green, if there aren't any `missingReviews`.

Reply:
Not really. `validatePullRequest` will only return reports for failed rules, but if no rule failed, it will return an empty array.

So, unless it failed, `generateCheckRunData` will be working with an empty array and returning a positive case.